android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/service/BdsService.java

```diff
@@ -111,6 +111,10 @@ public void makeRequest(String endPoint, String httpMethod, List<String> paths,
     AsyncTask asyncTask =
```

Comment:
since this is using generic types, you should use ServiceAsyncTask specific type instead of the generic AsyncTask